blog/2022-07-15-typescript-1.0.0.md

```diff
@@ -0,0 +1,157 @@
+> state in the event of failure—even if the machine goes offline or loses power in the middle of execution. We also
+> durably retry and timeout external calls, so if your services or databases are unreachable or returning errors, we
+> will keep retrying until they're back in a good state. This allows backend engineers to develop at a new, higher level
+> of abstraction that is oblivious to faults in the system. See also a [video
+> introduction](https://twitter.com/lorendsr/status/1544806504443695104?s=20&t=XFYNWSB8BEroAhjA6ATDPQ) and [formal
+> definition](https://docs.temporal.io/temporal).
+
+The SDK was designed with TypeScript-first developer experience in mind, but works equally well with JavaScript.
+
+## What makes this SDK unique?
+
+### Shared Core
+
+This is the first stable SDK built on top of a shared Rust Core SDK (see [blog
+post](https://docs.temporal.io/blog/why-rust-powers-core-sdk/) and [repo](https://github.com/temporalio/sdk-core/)). The
+development of the Core SDK was started roughly around the time we started to develop the TypeScript SDK, and the API
+boundary between the two mutually evolved over time.
+
+By investing in a shared Core, we can reuse a lot of the complex concurrency management and state machine logic to build
+new SDKs much faster. For example, our [Python SDK](https://github.com/temporalio/sdk-python) (currently in alpha) was
+built in just a few months. Sharing the logic makes all of our SDKs more reliable because when a problem is fixed in
+Core, it is fixed for all Core-based SDKs.
+
+### Sandboxed workflow runtime
+
+The SDK leverages V8 isolates (the technology behind Chrome's isolation) to run each Workflow in an isolated JavaScript
+runtime to get a distinct global scope and to prevent the use of "unsafe" JavaScript modules that could break the code's
+[deterministic constraints](https://docs.temporal.io/workflows#deterministic-constraints). All non-deterministic
+JavaScript APIs, such as getting the current time or a random number, have been replaced with [deterministic
+versions](https://docs.temporal.io/typescript/determinism#sources-of-non-determinism).
+
+With all of this put together, we eliminated an entire class of footguns and made it easier to get started with
+Temporal.
+
+## Built with the community
+
+When we released the first alpha version of the SDK back in March of 2021, we set out to iterate on the public API and
+have asked our community of users to help us shape it.
+
+We've been very fortunate to have received such wide adoption for the SDK, especially at such an early stage. The SDK's Slack channel
+([#typescript-sdk](https://temporal.io/slack)) has grown to more than 1000 members, and we've been actively responding
+to and supporting users on a daily basis.
+
+The feedback and trust we've gotten from our early adopters who put the SDK in production has been invaluable. We
```

Comment:
```suggestion
The feedback and trust we've gotten from our early adopters have been invaluable. We
```

Reply:
Fixed locally, should be `has been invaluable`.